src/algorithms/ctmrg.jl

```diff
@@ -117,200 +126,27 @@ function MPSKit.leading_boundary(envinit, state, alg::CTMRG)
                 @warn(
                     "CTMRG reached maximal number of iterations at (Δnorm=$Δnorm, ΔCS=$ΔCS, ΔTS=$ΔTS)"
                 )
-            return conv_condition, normnew, CSnew, TSnew, ϵ
+            flush(stdout)  # Flush output to enable live printing on HPC
+            flush(stderr)  # Same for @info, @warn, ...
```

Comment:
I think this is something we can handle with a logger as well. I don't have a lot of experience with working with HPC, but I would guess that by implementing a non-default logger and using that one, we don't have to pollute our functions with this.

Reply:
A specialized HPC logger would be a great solution, since I didn't want to pollute the main functions with it but it is quite necessary if you want to work in a HPC environment (which I currently do).

Reply:
I usually handle this by using a [`FormatLogger`](https://julialogging.github.io/reference/loggingextras/#LoggingExtras.FormatLogger) or a [`FileLogger`](https://julialogging.github.io/reference/loggingextras/#LoggingExtras.FileLogger-Tuple{Any}) from LoggingExtras.jl as the global logger, both of which have an `always_flush` option for live printing to `stderr`. This does not help with `stdout`, but since we probably anyway want to output all the verbosity as log messages and not plain print statements this might be enough.

Reply:
Thanks for the tip! I actually think a `FormatLogger` would be enough since with #51 all the `stdout` messages I previously needed are now also pushed into `stderr` via `@infov`. So that should solve the problem.